<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt XKB</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Layout</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Switching options</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XKB options</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="toolTip">
               <string>Antico will configure layouts and start layout indicator</string>
              </property>
              <property name="text">
               <string>&amp;Enable keyboard layouts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="toolTip">
               <string>Antico will start indicator but will use existing layout configuration</string>
              </property>
              <property name="text">
               <string>Indicator only</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="toolTip">
               <string>Antico will not configure or show keyboard layouts</string>
              </property>
              <property name="text">
               <string>Disable keyboard layouts</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpIndicatorOptions">
           <property name="title">
            <string>Indicator Options</string>
           </property>
           <layout class="QHBoxLayout" name="_2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="chkShowFlag">
              <property name="whatsThis">
               <string>Shows country flag on background of layout name in tray icon</string>
              </property>
              <property name="text">
               <string>Show country flag</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkShowSingle">
              <property name="text">
               <string>Show indicator for single layout</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpLayouts">
           <property name="whatsThis">
            <string>&lt;h1&gt;Keyboard Layout&lt;/h1&gt; Here you can choose your keyboard layout and model. The 'model' refers to the type of keyboard that is connected to your computer, while the keyboard layout defines &quot;which key does what&quot; and may be different for different countries.</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Keyboard &amp;model:</string>
              </property>
              <property name="buddy">
               <cstring>comboModel</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboModel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="whatsThis">
               <string>Here you can choose a keyboard model. This setting is independent of your keyboard layout and refers to the &quot;hardware&quot; model, i.e. the way your keyboard is manufactured. Modern keyboards that come with your computer usually have two extra keys and are referred to as &quot;104-key&quot; models, which is probably what you want if you do not know what kind of keyboard you have.
</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QHBoxLayout" name="_4">
                  <item>
                   <widget class="QLabel" name="textLabel1_4">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>1</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Available layouts:</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="buddy">
                     <cstring>srcTableView</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btnRemove">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTreeView" name="srcTableView">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectRows</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QToolButton" name="btnAdd">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="textLabel1_4_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                      <horstretch>3</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Active layouts:</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="buddy">
                     <cstring>dstTableView</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="_6">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QTreeView" name="dstTableView">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="alternatingRowColors">
                     <bool>true</bool>
                    </property>
                    <property name="selectionMode">
                     <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectRows</enum>
                    </property>
                    <property name="wordWrap">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="_7">
                    <item>
                     <widget class="QToolButton" name="btnUp">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QToolButton" name="btnDown">
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QHBoxLayout" name="_8">
              <item>
               <widget class="QLabel" name="textLabel1">
                <property name="text">
                 <string>Layout variant:</string>
                </property>
                <property name="buddy">
                 <cstring>comboVariant</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboVariant">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="whatsThis">
                 <string>Here you can choose a variant of selected keyboard layout. Layout variants usually represent different key maps for the same language. For example, Ukrainian layout might have four variants: basic, winkeys (as in Windows), typewriter (as in typewriters) and phonetic (each Ukrainian letter is placed on a transliterated latin one).
</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="textLabel1_6">
                <property name="text">
                 <string>Label:</string>
                </property>
                <property name="buddy">
                 <cstring>editDisplayName</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editDisplayName">
                <property name="maxLength">
                 <number>3</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="textLabel1_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Command:</string>
              </property>
              <property name="buddy">
               <cstring>editCmdLine</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="editCmdLine">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="whatsThis">
               <string>This is the command which will be executed to initialize configured layouts.</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="optionsFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QGridLayout" name="_9">
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Shortcuts for switching layout</string>
              </property>
              <layout class="QGridLayout" name="_11">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Main shortcuts:</string>
                 </property>
                 <property name="buddy">
                  <cstring>btnXkbShortcut</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="btnXkbShortcut">
                 <property name="whatsThis">
                  <string>This is a shortcut for switching layouts which is handled by X.org. It allows modifier-only shortcuts.</string>
                 </property>
                 <property name="text">
                  <string>None</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QToolButton" name="xkbClearButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>3d level shortcuts:</string>
                 </property>
                 <property name="buddy">
                  <cstring>btnXkbShortcut3d</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="btnXkbShortcut3d">
                 <property name="whatsThis">
                  <string>This is a shortcut for switching to a third level of the active layout (if it has one) which is handled by X.org. It allows modifier-only shortcuts.</string>
                 </property>
                 <property name="text">
                  <string>None</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QToolButton" name="xkb3dClearButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Alternative shortcut:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="btnXkbShortcutAlt">
                 <property name="text">
                  <string>None</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QToolButton" name="xkbAltClearButton">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="title">
               <string>Switshing policy</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QRadioButton" name="rdBtnSwitchGlobal">
                 <property name="text">
                  <string>&amp;Global</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdBtnSwitchPerDesktop">
                 <property name="text">
                  <string>Desktop</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdBtnSwitchPerApp">
                 <property name="text">
                  <string>Application</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rdBtnSwitchPerWin">
                 <property name="text">
                  <string>&amp;Window</string>
                 </property>
                </widget>
               </item>
              </layout>
              <zorder>rdBtnSwitchGlobal</zorder>
              <zorder>rdBtnSwitchPerApp</zorder>
              <zorder>rdBtnSwitchPerDesktop</zorder>
              <zorder>rdBtnSwitchPerWin</zorder>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1" rowspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_3"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox3_2">
           <property name="whatsThis">
            <string>Here you can set xkb extension options instead of, or in addition to, specifying them in the X11 configuration file.</string>
           </property>
           <property name="title">
            <string>Xkb Options</string>
           </property>
           <layout class="QGridLayout" name="_12">
            <item row="0" column="0" colspan="2">
             <widget class="QCheckBox" name="checkResetOld">
              <property name="text">
               <string>&amp;Reset old options</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QTreeView" name="xkbOptionsTreeView">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="textLabel1_3_2">
              <property name="text">
               <string>Command:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="editCmdLineOpt">
              <property name="whatsThis">
               <string>This is the command which will be executed to initialize configured options.</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
